joomla/joomla-cms
# #11234 fixed hide color issue

Pull Request for Issue #11234  .
#### Summary of Changes

Added css and class to change the color to red to maintain consistency. 
#### Testing Instructions
- Apply this PR.
  -Try to add/edit contact at administrator side 
- Goto "Display" tab.
- Change " Show Category" field options. in hide option, background is red color, and in another two options background is green color.
- Check Contact global configuration-> check same changes for "Contact Category" field.

Follow instructions of #11234
